src/output/tts/tts.py

```diff
@@ -4,7 +4,7 @@
 
 
 class TtsPatterns:
-    def __init__(self, online=True):
+    def __init__(self, online=True,):
```

Comment:
this looks wrong (and you seem to have only done this so that you can pass `quit` in `app.py` to this method call - even though `quit` is neither defined nor used)

Reply:
forgot to delete after an idea, thanks for the hint.